controller/homeRoutes.js

```diff
@@ -0,0 +1,82 @@
+const router = require('express').Router();
+const { User, Restaurant, Review } = require('../models');
+const withAuth = require('../utils/auth');
+
+router.get('/', async (req, res) => {
+  try {
+    // Get all Restaurants and JOIN with user data
+    const restaurantData = await Restaurant.findAll({
+    //   include: [
+    //     {
+    //       model: User,
+    //       attributes: ['name'],
+    //     },
+    //   ],
+    });
+
+    // Serialize data so the template can read it
+    const restaurants = restaurantData.map((restaurant) => restaurant.get({ plain: true }));
+
+    // Pass serialized data and session flag into template
+    res.render('homepage', { 
+      restaurants, 
+      logged_in: req.session.logged_in 
+    });
+  } catch (err) {
+    res.status(500).json(err);
+  }
+});
+
+router.get('/restaurant/:id', async (req, res) => {
+  try {
+    const restaurantData = await Restaurant.findByPk(req.params.id, {
+    //   include: [
+    //     {
+    //       model: User,
+    //       attributes: ['name'],
+    //     },
+    //   ],
```

Comment:
Same comment here about User. We might need to add a `model: Review` or something later.